nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -395,6 +395,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `nodePackages` package set now defaults to the LTS release in the `nodejs` package again, instead of being pinned to `nodejs-14_x`. Several updates to node2nix have been made for compatibility with newer Node.js and npm versions and a new `postRebuild` hook has been added for packages to perform extra build steps before the npm install step prunes dev dependencies.
 
-- boot.kernel.sysctl is defined as a freeformType and adds a custom merge option for "net.core.rmem_max" (taking the highest value defined to avoid conflicts between 2 services trying to set that value)
+- `boot.kernel.sysctl` is defined as a freeformType and adds a custom merge option for "net.core.rmem_max" (taking the highest value defined to avoid conflicts between 2 services trying to set that value).
```

Comment:
Spurious?

Reply:
Sorry, didn't understand here.

I just fixed the formatting there, no big changes.

Reply:
I will be a bit preciousist here: this fixup, being unrelated to mamt, should be relegated to another commit (or PR). It does nothing besides polluting the patch - and twice, here and in the generated XML.